project/project.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package project
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+// RelPath returns the dir relative to project's root.
+func RelPath(root, dir string) string {
+	d := strings.TrimPrefix(dir, root)
+	if d == "" {
+		d = "/"
+	}
+	return d
+}
+
+// AbsPath returns the absolute dir for project's root.
+func AbsPath(root, dir string) string {
```

Comment:
```suggestion
// AbsPath takes the root project dir and a dir path that is relative to the root project dir
// and returns an absolute path (relative to the host root).
func AbsPath(root, dir string) string {
```

Maybe, not sure if it is better =P. Since we have some relative paths that look like root (starting with /) seems interesting to be explicit about being root on the actual host.